abstract-interpretation.cabal

```diff
@@ -14,26 +14,30 @@ cabal-version:       >=1.10
 
 library
   exposed-modules:     Abstract.Configuration
+                     , Abstract.Environment
+                     , Abstract.Eval
                      , Abstract.Interpreter
                      , Abstract.Interpreter.Caching
-                     , Abstract.Interpreter.Dead
                      , Abstract.Interpreter.Collecting
+                     , Abstract.Interpreter.Dead
                      , Abstract.Interpreter.Symbolic
                      , Abstract.Interpreter.Tracing
                      , Abstract.Primitive
                      , Abstract.Set
                      , Abstract.Store
                      , Abstract.Syntax
+                     , Abstract.Term
                      , Abstract.Type
-                     , Abstract.Util
                      , Abstract.Value
                      , Control.Effect
+                     , Data.Functor.Classes.Show.Generic
```

Comment:
@tclem: It just occurred to me that this is gonna collide with any other package providing a module by the same name.

I think our best bet for the time being is to put this into an `other-modules` section for the library, which will prevent collisions.

Reply:
👍  Good call. I brought over the generic `Eq(1)` and `Ord(1)` work you've done as well and put them all in the `other-modules` section.